app/scrapers/ask.py

```diff
@@ -4,12 +4,10 @@
 
 class Ask:
     """Scrapper class for Ask"""
-    @classmethod
-    def __init__(cls):
+    def __init__(self):
```

Comment:
When you do not use __self__ in a method then it is right to decorate it with [@classmethod](https://docs.python.org/3/library/functions.html#classmethod) and various linters will flag it if you do not.